mesonbuild/interpreter/interpreter.py

```diff
@@ -2340,7 +2340,7 @@ def func_configure_file(self, node, args, kwargs):
             else:
                 mesonlib.dump_conf_header(ofile_abs, conf, output_format)
             conf.used = True
-        elif 'command' in kwargs:
+        elif kwargs['command']:
```

Comment:
And here we also don't know whether the user specified literally `command: []` but if they do, we apparently report it as "must specify one of these 3, no action provided".

Reply:
We can now change this too to `is not None`, like we did for configuration above.